Directory.Build.targets

```diff
@@ -31,11 +24,10 @@
     <!-- Unsafe code is required by PlatformDetection. -->
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
-  
-  <PropertyGroup Condition="'$(IsSourceProject)' == 'true'">
-    <GenerateDocumentationFile>false</GenerateDocumentationFile>
-    <DocumentationFile>$(MSBuildProjectDirectory)\$(AssemblyName).xml</DocumentationFile>
 
+  <PropertyGroup Condition="'$(IsSourceProject)' == 'true'">
+    <XmlPath>$(MSBuildProjectDirectory)\$(AssemblyName).xml</XmlPath>
```

Comment:
Please give this property a more descriptive name. This also makes sure that this won't break if the SDK would add such a property name.